tdrs-frontend/src/components/PrivateTemplate/PrivateTemplate.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect } from 'react'
+import PropTypes from 'prop-types'
+
+
+export default function PrivateTemplate({ title, children }) {
+
+    useEffect(() => {
+        document.title = `${title} - TDP - TANF Data Portal`
+    })
+
+    return (
+        <div className="grid-container margin-top-3" >
```

Comment:
Same class as the container that use to be repeated on each of the existing pages